docker-compose.yml

```diff
@@ -51,10 +51,12 @@ services:
       MONGO_INITDB_ROOT_PASSWORD: $MONGO_INITDB_ROOT_PASSWORD
       ME_CONFIG_BASICAUTH_USERNAME: $ME_CONFIG_BASICAUTH_USERNAME
       ME_CONFIG_BASICAUTH_PASSWORD: $ME_CONFIG_BASICAUTH_PASSWORD
-      MAINNET_PROVIDER: $MAINNET_PROVIDER
-      GOERLI_PROVIDER: $GOERLI_PROVIDER
-      KOVAN_PROVIDER: $KOVAN_PROVIDER
-      RINKEBY_PROVIDER: $RINKEBY_PROVIDER
+      ETHEREUM_MAINNET_PROVIDER: $ETHEREUM_MAINNET_PROVIDER
+      ETHEREUM_GOERLI_PROVIDER: $ETHEREUM_GOERLI_PROVIDER
+      ETHEREUM_KOVAN_PROVIDER: $ETHEREUM_KOVAN_PROVIDER
+      ETHEREUM_RINKEBY_PROVIDER: $ETHEREUM_RINKEBY_PROVIDER
+      CONFLUX_TESTNET_PROVIDER: $CONFLUX_TESTNET_PROVIDER
+      CONFLUX_MAINNET_PROVIDER: $CONFLUX_MAINNET_PROVIDER
```

Comment:
We can't afford having to specify providers here. This should proably on each feed's configuration or whatever, as in an ideal world, we should be able to add feeds without doing PRs, simply by modifying config files.

Reply:
We are adding it in this way because we are using providers with secrets instead of the Portainer's gateway. In other PR, we can avoid modifying this file and the provider's dictionary every time we want to support a new network.